### PR TITLE
Minor hygiene items

### PR DESCRIPTION
**Pull request by wesbiggs**
- open external links in a new tab (some via mdbook extension, others by hardcoding HTML)
- fix reference to deprecated public key announcement in PRId algorithm
- give a more useful internal link for assertionMethodPublicKeys in the Verifiable Credentials overview
- name items in the PRId description a little more consistently

### CODE REVIEW

**Review by shannonwells — APPROVED** [review]
Changes look good. Reviewed code only, did not download the branch.

**Comment by wesbiggs** [comment]
Ah, `jontze/action-mdbook` doesn't support the `mdbook-external-links` plugin. Will need to set up the Github action in a different way if we want to use that for running checks/deployments.

File: .github/workflows/verify-pr.yaml [hunk 1]
```diff
@@ -24,9 +24,7 @@ jobs:
         run: npm run lint
 
       - name: Set up mdBook 📚
-        uses: ./.github/workflows/common/set-up-mdbook
```
Comment (wilwade) [inline .github/workflows/verify-pr.yaml:27]: If you are removing this, then you can also remove `./.github/workflows/common/set-up-mdbook`

Comment (wesbiggs) [reply .github/workflows/verify-pr.yaml]: I changed the common script instead. Running cargo install is slower, but more aligned with local usage, and this repo is not so time-sensitive.

**Review by wilwade — APPROVED** [review]

**Pull request merged**